examples/threelang/web3gw/money.md

```diff
@@ -0,0 +1,19 @@
+!!web3gw.keys.define
+    tfc_mnemonic:'mom picnic deliver again rug night rabbit music motion hole lion where'
```

Comment:
I would choose tfgrid (preferably this one) or mnemonic here instead of tfc_mnemonic. People know they have one key for tfgrid and the concept of tfchain might be unfamiliar to them. 
For the stellar key i would use "stellar", for bitcoin "bitcoin", etc.